account_menu_invoice_refund/__init__.py

```diff
@@ -1,2 +1,3 @@
 # Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# © 2021 FactorLibre - Gerardo Gomez-Caminero <gerardo.gomez@factorlibre.com>
```

Comment:
Please remove copyright claims.
Copyright applies to original work author only, and porting the code is not enough to qualify for authorship.

Reply:
Removed